jetstream_pt/config.py

```diff
@@ -12,35 +12,80 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from jetstream.core.config_lib import ServerConfig
+import os
+import time
+
+import jax
+from absl import flags
 from jetstream_pt.engine import create_pytorch_engine
 
+FLAGS = flags.FLAGS
+
+
+def define_common_flags():
+  """Add common config flags to global FLAG."""
+  flags.DEFINE_string(
```

Comment:
just define those in the top level. and then whoever imports it will have that flags. see: https://source.corp.google.com/search?q=flags.DEFINE_string

Reply:
Ok, done. Intention was to optionally import all common flags from scripts